### PR TITLE
Bump deps to 1.2

### PR DESCRIPTION
**Pull request by pierDipi**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -15,16 +15,19 @@ require (
...
+	golang.org/x/net v0.0.0-20220225172249-27dd8689420f // indirect
+	golang.org/x/sys v0.0.0-20220227234510-4e6760a101f9 // indirect
 	google.golang.org/protobuf v1.27.1
 	k8s.io/api v0.22.5
 	k8s.io/apiextensions-apiserver v0.22.5
 	k8s.io/apimachinery v0.22.5
 	k8s.io/apiserver v0.22.5
 	k8s.io/client-go v0.22.5
 	k8s.io/utils v0.0.0-20211208161948-7d6a63dca704
-	knative.dev/eventing v0.30.1
```
Comment (matzew) [inline go.mod:25]: that's the 30 :see_no_evil:

File: go.mod [hunk 2]
```diff
@@ -15,16 +15,19 @@ require (
...
 	go.uber.org/zap v1.19.1
+	golang.org/x/crypto v0.0.0-20220214200702-86341886e292 // indirect
+	golang.org/x/net v0.0.0-20220225172249-27dd8689420f // indirect
+	golang.org/x/sys v0.0.0-20220227234510-4e6760a101f9 // indirect
 	google.golang.org/protobuf v1.27.1
 	k8s.io/api v0.22.5
 	k8s.io/apiextensions-apiserver v0.22.5
 	k8s.io/apimachinery v0.22.5
 	k8s.io/apiserver v0.22.5
 	k8s.io/client-go v0.22.5
```
Comment (matzew) [inline go.mod:26]: but this _has_ `0.22.5`

**Comment by matzew** [comment]
```
error: build error: error creating build container: initializing source docker://quay.io/openshift-knative/17-jdk-centos7:latest: Requesting bear token: invalid status code from registry 504 (Gateway Timeout)

```

/retest

**Comment by pierDipi** [comment]
/cc @matzew @skonto 

this helps bump dependencies in SO.

**Pull request merged**